README.md

```diff
@@ -141,6 +141,20 @@ allowedTags: [],
 allowedAttributes: []
 ```
 
+### "What if I want to set specific values on some attributes?"
```

Comment:
"Allow only" rather than "set" I think (we're not forcing any value to be set, but we're permitting certain values to be set, if desired).

Reply:
Done